addons/spreadsheet_dashboard_hr_expense/data/files/expense_dashboard.json

```diff
@@ -1132,38 +1143,50 @@
                 "measure": "total_amount",
                 "order": "desc"
             }
-        }
-    },
-    "pivotNextId": 9,
-    "lists": {
-        "1": {
-            "columns": [
-                "date",
-                "name",
-                "employee_id",
-                "payment_mode",
-                "activity_ids",
-                "analytic_account_id",
-                "company_id",
-                "attachment_number",
-                "total_amount_company",
-                "total_amount",
-                "state"
+        },
+        "9": {
+            "colGroupBys": [],
+            "context": {
+                "params": {
+                    "action": 952,
+                    "model": "hr.expense",
+                    "view_type": "pivot",
+                    "menu_id": 659,
+                    "cids": 1
+                }
+            },
+            "domain": [
+                [
+                    "sale_order_id",
+                    "!=",
+                    false
```

Comment:
`hr_expense` has no dependency to `sale`, see https://github.com/odoo/odoo/issues/103899